### PR TITLE
Update cross sbt versions + others

### PR DESCRIPTION
**Pull request by mkurz**

### CODE REVIEW

File: src/sbt-test/sbt-web-build-base/check-settings/test [hunk 1]
```diff
@@ -1,7 +1,7 @@
 > ^update
 
 # Make sure resolution was done for both sbt 0.13 and sbt 1.0
-$ exists target/scala-2.10/sbt-0.13/resolution-cache/com.typesafe.sbt/check-settings/scala_2.10/sbt_0.13/0.1-SNAPSHOT/resolved.xml.xml
-$ exists target/scala-2.12/sbt-1.0/resolution-cache/com.typesafe.sbt/check-settings/scala_2.12/sbt_1.0/0.1-SNAPSHOT/resolved.xml.xml
+$ exists target/scala-2.10/sbt-0.13/update/update_cache_2.10/output
+$ exists target/scala-2.12/sbt-1.0/update/update_cache_2.12/output
```
Comment (mkurz) [inline src/sbt-test/sbt-web-build-base/check-settings/test]: Because of coursier.

**Comment by mkurz** [comment]
@jroper I also switch to adoptopenjdk. Can you cut a new release? Thanks!

**Comment by jroper** [comment]
Published sbt-web-build-base 1.2.2.

**Comment by mkurz** [comment]
Great, thanks!

**Pull request merged**